rest/access/handlers.go

```diff
@@ -34,7 +44,7 @@ func Authenticate(state *state.State, r *http.Request, trustedCerts map[string]x
 	}
 
 	switch r.Host {
-	case state.Address().URL.Host:
+	case hostAddress:
```

Comment:
Should we do any validation here to check if `hostAddress` is an actual address, and not just an empty string?

Reply:
Yeah I suppose this will give a clearer error message if a downstream doesn't use the function correctly.

Reply:
Fixed.